### ISSUE TITLE
Prevent zooming on double-tap on mobile site

### ISSUE THREAD

**Issue by acagastya** [itaditya/trick-or-treat-game#63]
Rapidly tapping arrow keys on mobile browser zooms the page, which is not necessary, and makes it harder to play.

Can be resolved by 
```html
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no, user-scalable=0">
```

**Comment by itaditya** [comment]
Good Catch!

**Comment by itaditya** [comment]
To get issue closed automatically when PR merged put this in the commit `fixes #64`. Instead of `fixing #64`
